SegmentEditorFloodFilling/SegmentEditorFloodFillingLib/SegmentEditorEffect.py

```diff
@@ -75,28 +75,28 @@ def createCursor(self, widget):
     #return slicer.util.mainWindow().cursor
     return qt.QCursor(qt.Qt.PointingHandCursor)
 
-  def masterVolumeNodeChanged(self):
-    # Force recomputation of clipped master image data
-    self.clippedMasterImageData = None
+  def sourceVolumeNodeChanged(self):
+    # Force recomputation of clipped source image data
+    self.clippedSourceImageData = None
 
-    # Set scalar range of master volume image data to threshold slider
+    # Set scalar range of source volume image data to threshold slider
     import math
     import vtkSegmentationCorePython as vtkSegmentationCore
-    masterImageData = self.scriptedEffect.masterVolumeImageData()
-    if not masterImageData:
+    sourceImageData = self.scriptedEffect.sourceVolumeImageData()
```

Comment:
Keep backward compatibility with Slicer-5.0 to make it easier to keep the extension up-to-date for both stable and preview releases.